src/assets/translations/en/main.json

```diff
@@ -197,7 +197,8 @@
     "installed": "Installed",
     "others": "Others",
     "hardwareWallets": "Hardware Wallets",
-    "dustAmount": "Dust Amount"
+    "dustAmount": "Dust Amount",
+    "marketDataUnavailable": "Market data is temporarily unavailable for %{asset}"
```

Comment:
q: is temporarily the vernacular we want or do we prefer to do without it? It may not just be temporarily, and most likely won't be for assets that hit this